stubs/peewee/peewee.pyi

```diff
@@ -150,12 +150,12 @@ class _HashableSource:
     def __init__(self, *args, **kwargs) -> None: ...
     def alias(self, name) -> None: ...
     def __hash__(self) -> int: ...
-    def __eq__(self, other) -> bool: ...
-    def __ne__(self, other) -> bool: ...
-    __lt__: Incomplete
-    __le__: Incomplete
-    __gt__: Incomplete
-    __ge__: Incomplete
+    def __eq__(self, other) -> Expression | bool: ...
+    def __ne__(self, other) -> Expression | bool: ...
+    __lt__: Expression
```

Comment:
This seems incorrect, I assume `__lt__` should be a function returning an `Expression`, not an `Expression` itself.

Reply:
From peewee's source code here: https://github.com/coleifer/peewee/blob/2b2e876edf686f3f4ddf5f36df9b978f42504082/peewee.py#L909

Most of the comparisons return something from the `_e()` function:
```
    def _e(op):
        def inner(self, rhs):
            return Expression(self, op, rhs)
        return inner
    __lt__ = _e(OP.LT)
    __le__ = _e(OP.LTE)
    __gt__ = _e(OP.GT)
    __ge__ = _e(OP.GTE)

```

Reply:
Oops, thank you. Changed to `Callable[[Any], Expression]`